src/test/run-pass/issue-25180.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(std_misc)]
+
+const EMPTY: &'static Fn() = &|| ();
+
+const ONE_ARGUMENT: &'static Fn(u32) = &|y| assert!(y == 42);
```

Comment:
`assert!` still uses the formatting machinery. I'm curious if we prefer to disable pretty testing or just add feature gates. What do other tests do?